src/store-api/src/storage/requests.rs

```diff
@@ -29,4 +32,6 @@ pub struct ScanRequest {
     /// If set, it contains the amount of rows needed by the caller,
     /// The data source should return *at least* this number of rows if available.
     pub limit: Option<usize>,
+    /// The session timezone
+    pub timezone: Option<Arc<Timezone>>,
```

Comment:
Why `ScanRequest` need to be aware of time zone? Time zone is only needed to convert string literals to timestamp values, which should be done before scanning regions.

Also, I feel it's better to add a `Context` field instead of arranging timezone around projections and filters.

Reply:
For the `TimeRangePredicate`.

Reply:
The time range predicate uses `scalar_value_to_timestamp` to cast the `ScalarValue` to `Timestamp`, it must be aware of timezone. This will be done in #3175 .

https://github.com/GreptimeTeam/greptimedb/blob/cde5a36f5e7f97262daea85dcdee201d0c2323de/src/table/src/predicate.rs#L29C23-L29C48